datasets/pings.md

```diff
@@ -10,3 +10,48 @@
 
 You can find the reference documentation for all ping types
 [here](https://firefox-source-docs.mozilla.org/toolkit/components/telemetry/telemetry/concepts/pings.html).
+
+# Ping Metadata
+
+The telemetry data pipeline appends a `meta` key to the ping body containing
```

Comment:
I would phrase this more generally, that the pipeline appends certain metadata fields containing extra (or sometimes repeated) data, and that the [Dataset API](https://python-moztelemetry.readthedocs.io/en/latest/api.html#dataset) makes it available as a `meta` key. These same fields are also commonly made available in direct-to-parquet datasets with a `metadata` prefix as described [here](https://github.com/mozilla/firefox-data-docs/pull/191/files#diff-8a341d077298aca144a3bd72201f1958R30).